solvers/SE3MotionPlanningTree.m

```diff
@@ -0,0 +1,81 @@
+classdef SE3MotionPlanningTree < CompositeVertexArrayTree
+  properties
+    rbm
+    min_distance = 0.2;
+  end
+
+  methods
+    function obj = SE3MotionPlanningTree()
+      T_R3 = R3MotionPlanningTree();
+      T_SO3 = SO3MotionPlanningTree();
+      obj = obj@CompositeVertexArrayTree({T_R3, T_SO3},{'R3', 'SO3'});
```

Comment:
I felt like this better reflected the structure of the space. Also, it lets me reuse the distance metrics already defined in those classes.